release/downloadIstioCtl.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#
+# Downloader/Installer for Istioctl
+OS="$(uname)"
+if [ "x${OS}" = "xDarwin" ] ; then
+  OSEXT="osx"
+else
+  # TODO we should check more/complain if not likely to work, etc...
+  OSEXT="linux"
+fi
+
+if [ "x${ISTIO_VERSION}" = "x" ] ; then
+  ISTIO_VERSION=$(curl -L -s https://api.github.com/repos/istio/istio/releases/latest | \
+                  grep tag_name | sed "s/ *\"tag_name\": *\"\\(.*\\)\",*/\\1/")
+fi
+
+if [ "x${ISTIO_VERSION}" = "x" ] ; then
+  echo "Unable to get latest Istio version. Set ISTIO_VERSION env var and re-run. For example: export ISTIO_VERSION=1.0.4"
+  exit;
+fi
+
+NAME="istio-$ISTIO_VERSION"
+URL="https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/istio-${ISTIO_VERSION}-${OSEXT}.tar.gz"
+echo "Downloading $NAME from $URL ..."
+curl -L "$URL" | tar xz "$NAME/bin/istioctl"
```

Comment:
I was thinking of a way to create it without creating the directories.  Your choice.

```
curl -L "$URL" | tar xz --strip-components 2 "$NAME/bin/istioctl"
echo "Downloaded istioctl version $NAME .  Copy it into your path.
```

Reply:
/cc @esnible 

Ironically I was going to ask you about stripping off the container directories, so thanks. I'll amend. 👍